cypress/e2e/Test Cases/Test Case-8.cy.js

```diff
@@ -0,0 +1,20 @@
+// cypress/integration/responsiveLogin.spec.js
+
+describe('Responsive Login Test', () => {
+    it('Valid user can login when screen width is less than 1060px', () => {
+      // Set the viewport size to less than 1060px width
+      cy.viewport(1000, 800);
+  
+      // Visit the login page
+      cy.visit('https://www.saucedemo.com/');
+  
+      // Log in with a valid username and password
```

Comment:
Should be in the before hook
https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests
https://filiphric.com/cypress-basics-before-beforeeach-after-aftereach